<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusIdWidget</class>
 <widget class="QWidget" name="BusIdWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:black;
}

QWidget#widget{
	border:1px solid #fefe00;
	border-radius:6px;	
}
QLabel#label_LCUStatusInfo{
	font:75 16px;
	color:#fefe00;
}

/*******页码******/
QLabel#label_pageNum{
	color:white;
	font:16px;
	border:1px solid white;
	border-top-left-radius:4px;
	border-bottom-left-radius:4px;
	border-right:none;
}
/*****总页数*****/
QLabel#label_totalPage{
	color:white;
	font:16px;
	border:1px solid white;
	border-bottom-right-radius:4px;
	border-top-right-radius:4px;
	border-left:none;
}

QPushButton{
	color:#f9f900;
	font:75 16px;
	border:1px solid  #fafa00;
	border-radius:6px;
}

QPushButton:focus:!active{
	background:transparent;
}

QPushButton:pressed:active,QPushButton:focus:active{
	background:#0000fe;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_LCUStatusInfo">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>LCU状态信息</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>9</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>29</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>线号</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>接口灯孔</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>284</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_pageNum">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_totalPage">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>56</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>9</number>
            </property>
            <item>
             <widget class="QPushButton" name="pbt_pageUp">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>上一页</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbt_pageDown">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>27</height>
               </size>
              </property>
              <property name="text">
               <string>下一页</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>17</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
